python/ray/train/_internal/backend_executor.py

```diff
@@ -547,6 +550,7 @@ def initialize_session(
                 controller_actor_id=core_context.get_actor_id(),
                 datasets=datasets,
                 worker_group=self.worker_group,
+                start_time=self._start_time,
```

Comment:
nit: Please add a comment that describes how we are defining start time. Since this time will somewhere in between the original Trainer init/fit call, and when the training workers actually start running.

Reply:
Added a comment while we initialize this `self._start_time`.